docsource/architectureHighLevelDesign.rst

```diff
@@ -20,8 +20,10 @@ Data Flow
 
```

Comment:
If you redo the diagram again, strictly speaking the search api is a part of KBaseSearchEngine. I think there's 3 main parts:

The indexer - includes mongo, coodinator, workers
Data storage - elastic
search API

Not holding up the PR for that though.